test/lib/util/package_clause_generator_test.rb

```diff
@@ -20,7 +20,9 @@ def setup
         rpm.save!
       end
 
-      @content_view = FactoryGirl.build(:katello_content_view, :organization => organization, :repositories => [@repo])
+      @content_view = FactoryGirl.build(:katello_content_view, :organization => organization)
+      @content_view.save!
+      @content_view.repositories << @repo
```

Comment:
This test went from using a non-saved object to a database object -- curious why the change was needed?